ClassMoodApp/templates/liveView.html

```diff
@@ -0,0 +1,22 @@
+{% extends "layout.html" %}
+{% block body %}
+
+<div id="lectureId" lecture-id="{{lecture_id}}"></div>
```

Comment:
I'm guessing you are using this lecture-id in you ts file. Use ng-model instead so that u don't have to reference this div by the attribute.